package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "swatchjs",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
New backwards-compatible functionality means this should be a minor version bump.